report/main.Rmd

```diff
@@ -0,0 +1,313 @@
+
+
+
+```{r data, include=FALSE}
+beverage_sales <- read.csv("../rawdata/june1data.csv") %>% 
+   select(Count, DofW, Site, Intervention, ZeroCal, Sugary, Total)
+# head(beverage_sales)
+# dim(beverage_sales)
+```
+
+```{r Summarize Data, eval=FALSE, include=FALSE}
+# summarize_data(beverage_sales)$numerical
+# unique(beverage_sales$Site)
+# unique(beverage_sales$Intervention)
+```
+
+## 3. Exploratory Analysis
+
+Exploratory Data Analysis (EDA) is an essential precursor to formal statistical methods, uncovering underlying patterns and outliers in the data. It ensures the subsequent analysis is built on a solid foundation of understanding, thereby enhancing the reliability of the findings.
+
+
+### 3.1 Box Plots
+
+The following boxplots represent the distribution of sugary and zero-calorie beverage sales across different intervention strategies. Each boxplot captures the sales data variability with the central line denoting the median, the edges of the box indicating the interquartile range (IQR), and the whiskers extending to the furthest points that are not considered outliers. Outliers are individual points beyond the whiskers. Note that due to the bimodality of sales, the boxplot incorrectly indicates many outliers. The interventions, labelled on the x-axis, include baseline (preint), discount & messaging (dismes), washout period (wash), the discount only (dis), second washout (wash2), follow up (follow), calorie content poster (cal), the exercise required to burn the calorie content (excer), and combination of the two (both). 
+
+```{r Boxplots, fig.cap="The boxplots display the sales distribution of Sugary and Zero-Calorie beverages across various interventions. Each intervention is colour-coded and shows the range of sales data with the central line representing the median sales.", echo=FALSE, fig.height=3, fig.width=6, fig.pos="H"}
+boxplots_by_category(beverage_sales, responses = c("ZeroCal", "Sugary"), category = "Intervention")
+```
+
+
+### 3.2 Correlation Plot 
+  
+The following plot investigates the correlation structure between the day of the week (`DofW`), the number of zero-calorie drinks sold (`ZeroCal`), and the number of sugary drinks sold (`Sugary`). In this plot, the size, color of the circles and number represent the strength of the correlation coefficients between the variables. The `ZeroCal` and `Sugary` variables exhibit a very strong positive correlation with a correlation coefficient of 0.97. This suggests that sales of zero-calorie and sugary drinks are closely related; when sales of one type increase, sales of the other type tend to increase in a similar fashion. Conversely, both `ZeroCal` and `Sugary` drinks show a negative correlation with `DofW`, as indicated by the coefficient of -0.39. This negative correlation suggests a tendency for the sales of both drink types to decrease on certain days of the week.
```

Comment:
I believe considering the day of the week as an ordered variable may not be appropriate, as it's unclear if, for instance, Monday is less than Sunday. In this context, I would recommend excluding the day of the week from the correlation matrix. Additionally, it might be relevant to discuss some implications of these results.